cloudformation.yaml

```diff
@@ -26,29 +26,11 @@ Resources:
           - ExpirationInDays: 1
             Status: Enabled
             Prefix: "AWSLogs/"
-      VersioningConfiguration:
-        Status: Enabled
 
   ## The sqs queue we will deliver encrypted messages to
   SQSQueue:
     Type: 'AWS::SQS::Queue'
 
-  ## API Gateway Roles
```

Comment:
Are you sure this part should be removed? I don't remember adding that as part of the Python stuff.

Reply:
This was added 1 year ago in 8ab92a340c2a52f4742071856eb3786993066ec0 in the early iteration of the Python version. It's not present on the other Java-branch `cloudformation.yaml`s anywhere that I could find -- only `exercise-3-finished`. (Contrast `exercise-2-start` or `exercise-3-start`, for example.)

Looking through my Java deployment log history and other configs, it looks like this is the only glue we have right now connecting API Gateway to CloudWatch Logs. However, the log resolution we already have from e.g. Lambda logs looks good for workshop purposes. (This config is present on Python branches, so it would be a difference between the two.)

I think I'm inclined to get the Java branches all consistent again, and then test putting this connection back in on all branches in Java if we think it's net clarifying rather than confusing for the Java version. Thoughts?

Reply:
Fair enough. I suspect that was probably just me setting up APIG logging to help debug when I was building out the Python backend.